nautobot/core/views/mixins.py

```diff
@@ -131,6 +131,7 @@ class ObjectPermissionRequiredMixin(AccessMixin):
     """
 
     additional_permissions = []
+    use_new_ui = False
```

Comment:
Would you mind documenting this parameter in the class docstring?


I'm also not entirely comfortable with defining this in `ObjectPermissionRequiredMixin` at all - I get that it's probably our most widely-used mixin so that's convenient, but the UI toggle really doesn't have anything to do with permissions enforcement so it's kind of a mismatch. Should we add it to the generic view base classes directly instead?